primitives/transaction-pool/src/lib.rs

```diff
@@ -21,13 +21,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Please update module-level docs, cause it's just the runtime API now.

Perhaps worth to rename the crate?